governance/bot-config.json

```
@@ -0,0 +1,23 @@
+{
+  "developerAbbreviation": "AE",
+  "protocolName": "Aave",
+  "protocolAbbreviation": "AAVE",
+  "contracts": {
+    "AaveGovernanceV2": {
+      "address": "0xEC568fffba86c094cf06b22134B23074DFE2252c",
+      "abiFile": "AaveGovernanceV2.json",
+      "events": [
+        "ProposalCreated",
+        "ProposalExecuted",
+        "ProposalQueued",
+        "ProposalCanceled",
+        "VoteEmitted",
+        "ExecutorAuthorized",
+        "ExecutorUnauthorized",
+        "GovernanceStrategyChanged",
+        "OwnershipTransferred",
```

```suggestion
        {"OwnershipTransferred": {
          "alertType": "info",
          "alertSeverity": "suspicious"
          }},
```
An example of how you can handle different alert type for each event. As the others mentioned, the type and severity should be customizable.